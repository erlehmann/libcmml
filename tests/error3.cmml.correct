<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml>

<stream id="a1" basetime="5.000">
<import contenttype="audio/x-wav" src="00-3-18.wav" start="npt:0.000"/>
</stream>

<head lang="en">
<title>Buried dams of water</title>
</head>

