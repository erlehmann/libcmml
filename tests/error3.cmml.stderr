error3.cmml:24:0: Expat error parsing CMML filenot well-formed (invalid token)
